Add snowball token filter docs #8276 (#8399)
opensearch-project/documentation-website: _analyzers/token-filters/snowball.md

---
layout: default
title: Snowball
parent: Token filters
nav_order: 380
---

# Snowball token filter

> The `snowball` token filter is a stemming filter based on the [Snowball](https://snowballstem.org/) algorithm. It comes with support for many languages and offers improvement over the Porter stemming algorithm when it comes to efficiency and accuracy.

Reviewer: ```suggestion
The `snowball` token filter is a stemming filter based on the [Snowball](https://snowballstem.org/) algorithm. It supports many languages and is more efficient and accurate than the Porter stemming algorithm.
```
